window17onemoretime

2017-04-10--14-46-36

logging this instance of the experiment:


obtaining cookie from nvidia digits server

obtaining dataset from nvidia digits server
{

"commandline":
curl localhost:80/datasets/images/classification.json -b digits.cookie -XPOST -F folder_train=train--experiment0--window17-corrected--2017-04-03--17-55-07/ -F encoding=png -F resize_channels=1 -F resize_height=17 -F resize_width=17 -F method=folder -F dataset_name=window17onemoretime--2017-04-10--14-46-36

"returncode":
0

"stdout":
{
  "errors": {
    "folder_train": [
      "Folder does not exist"
    ]
  }
}

"stderr":
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   942  100    79  100   863  24360   259k --:--:-- --:--:-- --:--:--  421k


}
closing this instance with an exception:
<class 'KeyError'>
'id'
<traceback object at 0x7f6ca173f5c8>
